Makefile.am

```diff
@@ -49,6 +49,7 @@ endif
 
 # Shared data files
 install-data-hook:
+	mkdir -p /var/log/crmsh/; \
```

Comment:
This should belong to the `install-exec-local` part, and should be done the conventional way like:
`$(INSTALL) -d  -m 770 $(DESTDIR)/${localstatedir}/log/crmsh`

The macros are important for it to be created properly at the right place depending on build/installation environments, distributions and so on.

This should be also listed in the `%files` in crm.spec.in file as well like:

`%dir %attr (770, %{uname}, %{gname}) %{_var}/log/crmsh`

And please test it and make sure it works both for rpmbuild and manual build/install.

It might still be worth it to check if it exists at runtime and create it if it doesn't.

Reply:
> It might still be worth it to check if it exists at runtime and create it if it doesn't.

What do you think of this? :-) That's what you partially did in the original commit of this pull request.

Reply:
I have added a condition, that the file is only installed if it doesn't exist.

Reply:
By "runtime", I meant when crmsh is running, like what you did for in the original commit.

Reply:
:)

Reply:
I added the very first code. The Path function will create the /var/log/crmsh folder if it doesn't exist.